experimental/traffic-portal/src/app/core/profiles/profile-table/profile-table.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card class="table-page-content">
+	<div class="search-container">
+		<input type="search" name="fuzzControl" aria-label="Fuzzy Search Profiles" autofocus inputmode="search" role="search" accesskey="/" placeholder="Fuzzy Search" [formControl]="fuzzControl" (input)="updateURL()" />
+	</div>
+	<tp-generic-table
+		[data]="profiles | async"
```

Comment:
This is missing the dropdown from TPv1 that has `Import Profile` and `Compare Profile`. Currently tpv2 doesn't really have a method for adding that drop down but you can add more buttons through `[tableTitleButtons]`.

Compare Profiles should be disabled (needs parameters to be implemented), you'll probably have to modify the generic-table code slightly to allow for disabling of these buttons.

Reply:
Import profile i thought of doing when developing details page.
Compare profile after parameter is implemented, shall we add it later rather than adding and making it disabled.?

Reply:
That's fine, I usually like to add them there just to make sure it's not forgotten; given the rate at which we've been doing TPv2 stuff thats alright with me.